vignettes/projecting_incidence.Rmd

```diff
@@ -178,15 +178,6 @@ We will, therefore, run each simulation $100$ times.
 sim_rep <- 100
 ```
 
-Lastly, `chain_sim()` requires the maximum size of each chain. 
-Above this value, the simulation is cut off. If this value is 
-not specified, it assumes a value of infinity. Here, we will
-assume a maximum chain size of $1000$. We will call it `chain_threhold`.
-```{r}
-#' Maximum chain size allowed
-chain_threshold <- 1000
-```
-
```

Comment:
I've removed this because there was no rationale given for the maximum chain size (which would affect the projections). If it is to stay in I'd suggest such a rationale is added.